docs/en/operations/system-tables/information_schema.md

```diff
@@ -218,3 +226,62 @@ is_trigger_updatable:       NO
 is_trigger_deletable:       NO
 is_trigger_insertable_into: NO
 ```
+
+## KEY_COLUMN_USAGE (#key_column_usage)
+
+It was added for compatibility with third party tools such as Tableau Online. Contains only the primary keys columns read from [system.columns](../../operations/system-tables/columns.md).
```

Comment:
No need to mention Tableau Online. All of the views in information_schema were eventually added for compatibility with some 3rd party tool ... Therefore, let's rephrase a bit:

```md
Contains columns from the [system.tables](../../operations/system-tables/tables.md) system table which are restricted by constraints.
```